pint/utils.py

```diff
@@ -439,6 +440,74 @@ def is_number(s):
         pass
     return False
 
+def make_toas(startMJD, endMJD, ntoas, model, freq=1400, obs='GBT'):
+    '''make evenly spaced toas without errors'''
```

Comment:
Are these TOAs supposed to have residual zero? Mention also what values are acceptable for freq? In particular that you can generate multi-frequency TOAs by passing an array of different frequencies, but that you only allow one observatory.